### PR TITLE
Feat/home page new design

### PR DESCRIPTION
**Pull request by Darthmaul**
Implement new home page design, which includes some style changes discussed by the group previously.

This tackles the implementation of this concept: https://trello.com/c/JK19vswx/49-dashboard-home-page-re-work

Users may have some feedback regarding this page, but realistically this will give us a more concise page with faster speeds.

### CODE REVIEW

File: app/Http/Controllers/DashboardController.php [hunk 1]
```diff
@@ -64,9 +62,16 @@ private function getLatestLoadouts()
...
-            ->take(12)
+            ->take(3)
             ->get();
 
         return $latestLoadouts;
     }
+
+    private function getLatestPosts()
+    {
+        $latestPosts = Post::orderBy('created_at', 'desc')->take(3)->get();
```
Comment (chaseconey) [inline app/Http/Controllers/DashboardController.php:73]:
```suggestion
        $latestPosts = Post::latest()->take(3)->get();
```

File: resources/views/dashboard/index.blade.php [hunk 2]
```diff
@@ -2,29 +2,64 @@
 
 @section('body_class', 'body__home')
 
-@section('content')
-
-    <call-to-action></call-to-action>
+@section('styles')
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@splidejs/splide@latest/dist/css/splide.min.css">
```
Comment (chaseconey) [inline resources/views/dashboard/index.blade.php:6]: Doesn't look like we are using this, right?

File: resources/views/dashboard/index.blade.php [hunk 3]
```diff
@@ -2,29 +2,64 @@
...
+                <x-dashboard-listing :loadoutList="$latestLoadouts" title='Newest Loadouts' />
             </div>
-            @endforeach
         </div>
     </div>
 
+    <div class="featuredLoadoutsContainer px-3 pb-2">
+        @foreach ($latestPosts as $post)
+            <div class="my-5">
+                <a href="/blog/{{$post->id}}" class="hover:underline"><h3>{{$post->title}}</h3></a>
```
Comment (chaseconey) [inline resources/views/dashboard/index.blade.php:47]: Should use the `route` function here.

Something like `href="{{ route('blog.show', $post->id) }}"`

File: resources/views/dashboard/index.blade.php [hunk 4]
```diff
@@ -2,29 +2,64 @@
...
+                <p class="dashPost">{!! Str::words($post->content, 50, '...') !!}</p>
+                <a class="dashButton px-3 w-4 ml-0 mt-1" href="/blog/{{$post->id}}">
```
Comment (chaseconey) [inline resources/views/dashboard/index.blade.php:49]: Same here.

**Review by chaseconey — CHANGES_REQUESTED** [review]

File: resources/views/dashboard/index.blade.php (hunk 2 above)
Comment (Darthmaul) [reply resources/views/dashboard/index.blade.php:6]: Yep removing thanks

File: resources/views/dashboard/index.blade.php (hunk 3 above)
Comment (Darthmaul) [reply resources/views/dashboard/index.blade.php:47]: Bada bing

File: resources/views/dashboard/index.blade.php (hunk 4 above)
Comment (Darthmaul) [reply resources/views/dashboard/index.blade.php]: Bada bing

File: app/Http/Controllers/DashboardController.php (hunk 1 above)
Comment (Darthmaul) [reply app/Http/Controllers/DashboardController.php:73]: Bada boom

**Review by chaseconey — APPROVED** [review]

**Pull request merged**